src/DB_Command.php

```diff
@@ -1007,6 +1030,7 @@ public function size( $args, $assoc_args ) {
 			$rows[] = [
 				'Name' => DB_NAME,
 				'Size' => strtoupper( $db_bytes ) . $default_unit,
+				'_Bytes' => strtoupper( $db_bytes ),
```

Comment:
Is there a reason we need to add an underscore to `_Bytes`? I think it's fine to include in the output...

Reply:
The fields are configured above that line to only output `Name` and `Size`, so I added an `_` to the `Bytes` temporary key as a convention for visibility. It doesn't change the code either way, so it can be easily removed if you'd like.

Reply:
Cool, let's go ahead and remove the underscore. Thanks!

Reply:
Resolved by https://github.com/wp-cli/db-command/pull/226/commits/8ba0f6aae72e2d1bf17e147c727935478ef2aab3.